### PR TITLE
Feat: new about

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR includes:
- About page MVP. There's still plenty to do like:
    - About sections (Team, Partners, Funders).
    - Rest of static pages.
    - Nav sidebar subsections (with separator).

But I figured better to approach them in other PRs and not bloat this one (already 29 files long).

### CODE REVIEW

File: frontend/html/about-old.ejs [hunk 1]
```diff
@@ -0,0 +1,237 @@
+<div class="loading-wrapper">
```
Comment (tiagojsag) [inline frontend/html/about-old.ejs]: is this file needed?

File: frontend/scripts/pages/about-old.page.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+/* eslint-disable no-new */
+import AboutMarkup from 'html/about-old.ejs';
```
Comment (tiagojsag) [inline frontend/scripts/pages/about-old.page.jsx]: and this?

File: frontend/scripts/index.ejs [hunk 3]
```diff
@@ -6,6 +6,7 @@
 <%= htmlWebpackPlugin.options.icons %>
 
 <div id="app-root-container"></div>
+<div id="app-portal-container"></div>
```
Comment (tiagojsag) [inline frontend/scripts/index.ejs]: Don't really see the point of using a portal here. Are you sure this is the best option?

File: frontend/scripts/react-components/shared/portal.component.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```
Comment (tiagojsag) [inline frontend/scripts/react-components/shared/portal.component.jsx]: is it me, or this is not actually used?

File: frontend/scripts/router.js [hunk 5]
```diff
@@ -65,35 +70,45 @@ const routes = {
...
     path: '/FAQ',
-    page: 'FAQ',
-    extension: 'jsx'
+    page: 'static-content',
+    extension: 'jsx',
+    thunk: dispatchThunks(getPageStaticContent)
   },
   [NOT_FOUND]: {
-    path: '/404'
+    path: '/404',
```
Comment (tiagojsag) [inline frontend/scripts/router.js]: Please create a separate issue for us to test if this is actually returning an 404 http code

File: frontend/scripts/react-components/static-content/static-content.component.jsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+import NavSidebar from './nav-sidebar.component';
+
+class StaticContent extends React.PureComponent {
+  render() {
+    const { links = [], content = '', children, notFound } = this.props;
+    const MarkdownContainer = p => (<div className="markdown-content">{p.children}</div>);
+    return (
+      <div className="c-static-content">
+        <Hero className="-read-only" />
+        {!notFound &&
```
Comment (tiagojsag) [inline frontend/scripts/react-components/static-content/static-content.component.jsx]: Kudos for not using trenary operator here!!!!

Comment (tiagojsag) [reply frontend/scripts/react-components/static-content/static-content.component.jsx]: This should probably be two separate components.

File: frontend/scripts/react-components/static-content/static-content.actions.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+export const getStaticContentFilename = location => camelCase(
+  `${location.type} ${(location.payload.section || '')}`
+);
+
+export const getStaticContent = () => (dispatch, getState) => {
+  const { location, staticContent } = getState();
+  const filename = getStaticContentFilename(location);
+  const notFound = location.type === NOT_FOUND;
+  if (
+    typeof staticContent.markdown[filename] === 'undefined' &&
```
Comment (tiagojsag) [inline frontend/scripts/react-components/static-content/static-content.actions.js]: wouldn't `staticContent.markdown[filename] === undefined` be equivalent? just asking, no need to change if they're the same

File: frontend/scripts/react-components/static-content/static-content.actions.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    !notFound
+  ) {
+    const url = `/static-content/${filename}.md`;
```
Comment (tiagojsag) [inline frontend/scripts/react-components/static-content/static-content.actions.js]: why not use my awesome beautiful getUrl class? https://github.com/Vizzuality/trase/blob/94f3270dfc54f9b79cc9fbfe7c1cc802aac5e93f/frontend/scripts/utils/getURLFromParams.js

File: frontend/scripts/react-components/static-content/static-content-links.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
+
+export default {
```
Comment (tiagojsag) [inline frontend/scripts/react-components/static-content/static-content-links.js]: I'm wondering if moving this into router.js makes sense. Seems like the first place us-in-6-months will look for when we need to add more pages. What do you think?

Comment (sorodrigo) [reply frontend/scripts/react-components/static-content/static-content-links.js]: I actually struggled deciding where to put this. If we put it in the router, we should split that file.

Comment (sorodrigo) [reply frontend/scripts/react-components/static-content/static-content-links.js]: Actually I decided against this, the links have more sense in the `static-content` context. If you take them up to the router it will be confusing. Besides, it's currently a relative import so following the breadcrumbs won't be hard. 🍪

File: frontend/scripts/react-components/static-content/static-content.component.jsx (hunk 6 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/static-content/static-content.component.jsx]: I knew you'd like it. Food for thought. https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340

File: frontend/scripts/react-components/static-content/static-content-links.js (hunk 9 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/static-content/static-content-links.js]: In the end when the new nav, refactoring this made more sense. I made this change on the new nav PR

File: frontend/scripts/react-components/static-content/static-content.component.jsx (hunk 6 above)
Comment (tiagojsag) [reply frontend/scripts/react-components/static-content/static-content.component.jsx]: Hum I don't really agree with that article. The author basically advocates that trenary operators are great because they are less verbose and clutered than if/else/return statements, which they are, but that does not make it easier for humans to understand and debug. Humans many times need help from those "noisy" operators, otherwise we'd all be programming in https://en.wikipedia.org/wiki/Whitespace_(programming_language)

**Comment by tiagojsag** [comment]
Seems we have some homepage css regressions introduced in this PR :/ otherwise LGTM

File: frontend/scripts/react-components/static-content/static-content.component.jsx (hunk 6 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/static-content/static-content.component.jsx]: hahaha 👍

**Comment by sorodrigo** [comment]
ready
